### ISSUE TITLE
matrix.to event permalink doesn't jump to event

### ISSUE THREAD

**Issue by lub** [vector-im/riotX-android#1538]
It seems permalinks to events (e.g. https://matrix.to/#/!xYvNcQPhnkrdUmYczI:matrix.org/$M5QAmlcBXdS2MVV26MHtJzFITOikr4l3EXaRh1-T018?via=bpulse.org&via=matrix.org&via=privacytools.io) open the correct room, but don't scroll to or highlight the actual event

**Comment by richvdh** [comment]
I *think* this is limited to events that have subsequently been edited, and that RiotX jumps to the point in the timeline that the (invisible) edit event would be.

**Comment by bmarty** [comment]
For not editied event, I think the event is not highlighted, but the scroll should work. This has to be implemented though.

For edited event we have to check

**Comment by lub** [comment]
Nah, this not only happens with edited events. I agree it does some jumping (now that there are some messages after the event I see that), but it looks completely random.

The events are just random examples from the TWIM room, sorry if I sparked some confusion with that :D

For example the next message (https://riot.im/app/#/room/!xYvNcQPhnkrdUmYczI:matrix.org/$sb1xpJakufMEfHCAmeP0E69FyEsbziU6OIQxCmQwNTU) is not edited. This is what I get in riot-web:

![image](https://user-images.githubusercontent.com/1029578/85714579-da57a180-b6ea-11ea-815a-a12ada08378c.png)

riotx-android instead scrolls to messages three hours later:
![image](https://user-images.githubusercontent.com/1029578/85714661-ee9b9e80-b6ea-11ea-93fe-bae5e4eb0570.png)
